components/Collections/examples/CollectionsEditingExample.m

```diff
@@ -87,6 +92,32 @@ - (UICollectionViewCell *)collectionView:(UICollectionView *)collectionView
   return cell;
 }
 
+- (UICollectionReusableView *)collectionView:(UICollectionView *)collectionView
+           viewForSupplementaryElementOfKind:(NSString *)kind
+                                 atIndexPath:(NSIndexPath *)indexPath
+{
+
+  // Must include below code snippet
+  id supplementaryView = [super collectionView:collectionView
+             viewForSupplementaryElementOfKind:kind
+                                   atIndexPath:indexPath];
+  if (supplementaryView) {
```

Comment:
MDCCollectionView injects a private supplementary view of kind MDCCollectionInfoBarView. As this class is private, and the super class of MDCCollectionViewController calls the collectionView:viewForSupplementaryElementOfKind:atIndexPath: itself, any subclass that implements this method has to either re-implement this method or somehow get this view from its super class. As far as whether it comes first or fallback, I am thinking this would be easier to implement from an outside user's standpoint. Thoughts?